packages/docs/src/styleguide.md

```diff
@@ -19,7 +19,7 @@ npm install --save @blueprintjs/core react react-dom react-addons-css-transition
 Import components from the `@blueprintjs/core` module into your project.
 Don't forget to include the main CSS stylesheet too!
 
-**[See Components Usage for more complete installation instructions.](#components.usage)**
+[See **Components usage** for more complete installation instructions.](#components.usage)
```

Comment:
it's more like `Components > Usage` (breadcrumbs) so capitalization makes sense for both words. i wouldn't be opposed to adding the `>` even.